app/frontend/src/stories/__mocks__/service.ts

```diff
@@ -1,44 +1,10 @@
-import { MessageProps } from "../../features/messages/messagelist/MessageView";
-import { HostingStatus, User } from "../../pb/api_pb";
-import { HostRequestStatus, Message } from "../../pb/conversations_pb";
-import { PageType } from "../../pb/pages_pb";
 import { service as originalService } from "../../service";
-import funnycat from "../assets/funnycat.jpg";
-import funnydog from "../assets/funnydog.jpg";
-import funnykid from "../assets/funnykid.jpg";
+import messages from "../../test/fixtures/messages.json";
+import users from "../../test/fixtures/users.json";
+import getThreadRes from "../../text/fixtures/getThreadRes.json";
+import groupChat from "../../text/fixtures/groupChat.json";
 
-export const user1 = {
-  name: "Funny Cat current User",
-  userId: 1,
-  username: "funnycat",
-  avatarUrl: funnycat,
-  age: 28,
-  city: "Los Angeles",
-  hostingStatus: HostingStatus.HOSTING_STATUS_CAN_HOST,
-  communityStanding: 0.5,
-  verification: 0.5,
-  aboutMe: "I am a user",
-} as User.AsObject;
-
-const user2 = {
-  ...user1,
-  name: "Funny Dog",
-  userId: 2,
-  username: "funnydog",
-  avatarUrl: funnydog,
-} as User.AsObject;
-
-const user3 = {
-  ...user1,
-  name: "Funny Kid",
-  userId: 3,
-  username: "funnykid",
-  avatarUrl: funnykid,
-} as User.AsObject;
-
-const userMap = new Map(
-  [user1, user2, user3].map((user) => [user.userId, user])
-);
+const userMap = new Map(users.map((user) => [user.userId, user]));
```

Comment:
Hope the conflict I'll get here won't be too hard to fix... this map thing is driving me nuts since it doesn't work with username to user (I need that for the user's page stuff 😠 )

Reply:
Can you just change the mock so if it the key doesn't parse as a number, it does a username lookup?

Reply:
I ended up changing it back to a regular object so the mock just do a look up on this. It's only 3 users so not that bad to write it out in full for both IDs and usernames.